### ISSUE TITLE
License for the ostrich video in the docs?

### ISSUE THREAD

**Issue by RobRoseKnows** [AdamSpannbauer/python_video_stab#19]
Hey, is the Ostrich video used as an example owned by you and released under MIT as well?

**Comment by AdamSpannbauer** [comment]
Thanks for this.  I clipped the video early on in dev process, and hadn't stopped to rethink it's licensing.

The clip is from [Lion Country Safari - Drive-Thru Safari](https://www.youtube.com/watch?v=9pypPqbV_GM) from the channel [Happy Living](https://www.youtube.com/channel/UCFG_hvK09O_VLhw6hINSB9g).  I've reached out to the creators to ask permission to use the clip, and how to appropriately credit.  If I don't hear back soon or if they deny permission I will remove the clip and replace it with a different example.
